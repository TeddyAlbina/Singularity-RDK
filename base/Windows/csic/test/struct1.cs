public class Test {
	static char cs='G';
	public char ci;

	public Test(char x) {
		ci=x;
	}

	public static void Main() {
		char c='D';
		System.Console.WriteLine(c);
		System.Console.WriteLine(c.ToString());
		char[] ca=new char[] { 'F', 'H'};
		System.Console.WriteLine(ca[1].ToString());
		System.Console.WriteLine(cs.ToString());
		Test t=new Test('B');
		System.Console.WriteLine(t.ci.ToString());
	}
}
