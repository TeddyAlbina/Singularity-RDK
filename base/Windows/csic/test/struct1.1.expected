D
D
H
G
B
